src/cloud/docker/docker-quick-reference.md

```diff
@@ -37,17 +37,22 @@ Use the `-f` argument to specify an alternate configuration file. The following
 docker-compose -f docker-compose.yml -f docker-compose-custom.yml [-f more-custom-docker-compose.yml] up
 ```
 
-### Build options
+## Docker compose configuration generator
+
+{%include cloud/cloud-docker-config-generator-cmds.md%}
+
+### Additional build options
 
 | Option       | Key              | Available values
 | ------------ | ---------------- | ------------------
 | [Mode][]         | `--mode`, `-m`   | production, developer
 | [File synchronization engine][] | `--sync-engine` | native (default), docker-sync, mutagen
+| Specify a custom URL for Magento | `--host`<br>`--port`
 
 {:.bs-callout-info}
-See [Service versions] for a list of the options to configure and customize software service versions and other options when building your {{site.data.var.mcd-prod}} environment.
+See [Service versions] for a list of service configuration options you can add to the `ece-docker build:compose` command to customize the Cloud Docker environment configuration.
 
-## bin/magento-docker
+## bin/magento-docker CLI commands
```

Comment:
The original intention of this page was a quick reference or cheat sheet starting with docker-compose. It is best to keep title very brief, one word, if possible. If you add "CLI commands" here, I think it loses the spirit since the whole page should be about commands. But if you do, then you need to add it in the other headings (`docker-compose` for one) for consistency.